tests/foreman/api/test_reporttemplates.py

```diff
@@ -24,18 +25,17 @@
 from robottelo.api.utils import promote
 from robottelo.api.utils import upload_manifest
 from robottelo.constants import DEFAULT_SUBSCRIPTION_NAME
-from robottelo.constants import DISTRO_RHEL7
 from robottelo.constants import PRDS
 from robottelo.constants import REPOS
 from robottelo.constants import REPOSET
+from robottelo.datafactory import parametrized
 from robottelo.datafactory import valid_data_list
-from robottelo.test import APITestCase
+from robottelo.hosts import ContentHost
 from robottelo.utils.issue_handlers import is_open
-from robottelo.vm import VirtualMachine
 
 
-@pytest.fixture(scope='class')
-def setup_content(request):
+@pytest.fixture(scope='module')
+def setup_content():
     org = entities.Organization().create()
```

Comment:
Consider using `module_org` fixture to supply the organization entity, this consolidates our fixture use.

Reply:
`module_lce` fixture can also supply the LifecycleEnvironment entity, with the `module_org`.